src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Members.cs

```diff
@@ -678,9 +678,13 @@ public override void VisitParameter(IParameterSymbol symbol)
             // (e.g. field types, param types, etc), which just want the name whereas parameters are
             // used on their own or in the context of methods.
 
-            var includeType = format.ParameterOptions.IncludesOption(SymbolDisplayParameterOptions.IncludeType);
+            // SignatureOnlyParameterSymbol.ContainingSymbol throws.
+            var isFunctionPointerParameter = symbol is not Microsoft.CodeAnalysis.CSharp.Symbols.PublicModel.Symbol { UnderlyingSymbol: SignatureOnlyParameterSymbol }
```

Comment:
```csharp
symbol is not Microsoft.CodeAnalysis.CSharp.Symbols.PublicModel.Symbol { UnderlyingSymbol: SignatureOnlyParameterSymbol }
```

This is horrible - would it be possible to make `SignatureOnlyParameterSymbol` return null instead of throwing @333fred?

Reply:
Not easily. It would require a significant refactoring of construction.